common/views/components/Promo/README.md

```diff
@@ -0,0 +1,44 @@
+# Promos
+What's their purpose, what they aren't, and what they have the potential to become.
+
+A promo is how [content types from our domain](https://user-images.githubusercontent.com/31692/35921561-dc6e90bc-0c12-11e8-96c5-46a01573afd1.png) promote themselves across our sites, and externally, based on a combination of content, promotional, and contextual data.
+
+The purpose of a promo is to surface content that might be relevant to you, or your current context within our site, and allow you to decide whether you'd like to explore that content further whether it be through interest or curiosity.
+
+It's more obvious purpose is to give you a link to that content 🚀.
+
+![An example of a promo using contextual, content, and promotional data](https://user-images.githubusercontent.com/31692/35923356-ea0a0030-0c17-11e8-8e99-4d28412b73cb.png)
+
+## Using the above example:
+
+*   The "5" is contextual as this promo is in a list of ordered articles
+*   The series title is directly linked to the content of the article
+*   The text below the image has been created by editors to promote the article. It is not actually part of the article's body.
+
+## What they aren't
+
+Promos are not adverts, nor functional components.
+
+![An image and text letting visitors know that there is a special menu at the Wellcome Kitchen](https://user-images.githubusercontent.com/31692/35924333-aa19db3c-0c1a-11e8-9451-36801b307bd3.png)
+
+The above is not a promo, but rather an ad for a special menu at Wellcome Collection's kitchen
+
+![{`An](https://user-images.githubusercontent.com/31692/35924690-a1f9830c-0c1b-11e8-89c0-964eaeb397ed.png)
+
+The above is not a promo either, but rather an enticing newsletter signup component
+
+## Potentialities
+
+We would like to look at the difference between pull and push promos. i.e. content that we surface and allow you to choose from vs content we have deemed to be of special interest and thus promote more emphatically.
+
+Currently we only have image promos, we'd like to explore using more specialised promos for image galleries, videos, audio etc.
+
+## Current promo types
+
+We currently have 5 promos.
+
+*   Editorial promos
+*   Event promos
+*   Exhibition promos
+*   Works promos
+*   Book Promos
```

Comment:
Maybe we can just leave this out so we don't have to update it.
We have the list in the story.

Reply:
👍